docs/content/foundation/appearance/border.mdx

```diff
@@ -4,10 +4,14 @@ nav_label: Border
 nav_order: 20
 ---
 
-# NCIDS Border
+# Border
 
-Nulla facilisis vitae lectus quis ultricies. Aliquam sodales scelerisque sapien et dictum.
-Mauris eu nunc sit amet lorem blandit gravida et ac dui. Suspendisse accumsan eros sit
-amet arcu lobortis, in consequat nunc molestie. Aenean lobortis molestie convallis.
-Nullam sit amet turpis vel nisi tristique dapibus sed in libero.
-Vivamus ac tristique nisl, eu bibendum eros.
+Set width, color, style, and radius of an item’s borders.
+
+## Borders in USWDS
+
+Border utilities have not been modified from USWDS. Refer to [USWDS Border Utilities](https://designsystem.digital.gov/utilities/border/).
```

Comment:
for this page please add the border color tables from https://github.com/NCIOCPL/ncids/wiki/NCIDS-Utility

and show example mixin (see https://designsystem-dev.cancer.gov/pr-1248/foundation/color - color utilities)

Reply:
Wasn't this approved by product? Why are we making content additions post-approval?

Reply:
Updated regardless

Reply:
could you remove the "False" ones since they really cant do anything with them, sorry I didnt say that before.